packages/swirl-components/src/components/swirl-separator/swirl-separator.tsx

```diff
@@ -20,10 +20,25 @@ export type SwirlSeparatorSpacing =
   tag: "swirl-separator",
 })
 export class SwirlSeparator {
-  @Prop() borderColor?: SwirlSeparatorBorderColor = "default";
+  @Prop() borderColor?: SwirlSeparatorColor = "default";
+  @Prop({ mutable: true }) color?: SwirlSeparatorColor = "default";
+  @Prop() label?: string;
```

Comment:
**suggestion**: it would be nice to control the label size.

- we could to support a new `labelSize` property (same used in `SwirlText`)
- and in case more control is required, add a new `labelElement` property (instead of `label/labelSize` properties)

Reply:
My ideia was to be a bit more strict with the design, and since from a design system perspective we will most likely not have the possibility to have different label sizes, I went with this simpler approach for now. In the future if we have the need to have different label sizes, we could also extend.

If you have strong opinions, we can discuss it with the designers :)